src/devices/ovrheadset/OVRHeadset.h

```diff
@@ -126,6 +126,7 @@ class OVRHeadset : public yarp::dev::DeviceDriver,
     std::map<ovrResult, std::string> error_messages;
 
     IFrameTransform* tfPublisher;
+    bool             relative;
```

Comment:
Is there a way to set this or is this just fixed to false in the constructor? Perhaps it should be an option...